src/apis/tokenlistapi.py

```diff
@@ -0,0 +1,50 @@
+"""
+TokenListAPI for fetching a curated token list.
+"""
+# pylint: disable=logging-fstring-interpolation
+from typing import Optional
+import requests
+from src.helper_functions import get_logger
+from src.constants import (
+    header,
+    REQUEST_TIMEOUT,
+)
+
+
+class TokenListAPI:
+    """
+    Class for fetching a curated token list.
+    """
+
+    def __init__(self) -> None:
+        self.logger = get_logger()
+
+    def get_token_list(self) -> Optional[list[str]]:
+        """
+        Returns a token list.
+        """
+        kleros_url = "http://t2crtokens.eth.link"
+        one_inch_url = "https://tokens.1inch.eth.link"
+        aave_url = "https://tokenlist.aave.eth.link"
+
+        url_list = [kleros_url, one_inch_url, aave_url]
+
+        token_list: list[str] = []
+        for url in url_list:
+            try:
+                data = requests.get(
+                    url,
+                    headers=header,
+                    timeout=REQUEST_TIMEOUT,
+                )
+                rsp = data.json()
+                if "tokens" in rsp:
+                    for token in rsp["tokens"]:
+                        token_list.append(token["address"].lower())
+            except requests.RequestException as err:
+                self.logger.warning(
+                    f"Connection error while fetching a token list, error: {err}"
+                )
```

Comment:
This conforms to our current convention. Potentially we should switch to something like the following:
```suggestion
                self.logger.warning(
                    f"Exception while fetching a token list: {err}"
                )
```
Otherwise an alert is triggered in the slack channel due to the word "error" being part of the logged message.

At some point we should switch to a more systematic approach for triggering alerts.

Reply:
Are you sure about that? Don't the slack alerts only have to do with the level of logging we use (error,warning,info)?

Reply:
The query run on logs is `stream: stderr AND kubernetes.namespace_name: ebbo AND log: ERROR`. So it does check for the word `error` (disregarding capitalization).

Reply:
Oh ok. And the warning msgs are sent to stderr? So that all 3 conditions are indeed satisfied?

Reply:
At the moment all messages are sent to stderr. Only sending some levels to stderr is possible but requires changing the logger.

Maybe we can just create an issue and do that later.